plugin/pkg/admission/resourcequota/controller.go

```diff
@@ -169,6 +172,11 @@ func (e *quotaEvaluator) checkAttributes(ns string, admissionAttributes []*admis
 		return
 	}
 
+	if e.lockAquisitionFunc != nil {
```

Comment:
we should get a test in the future to ensure that this is respected and not lost in future refactors.